src/Native/Unix/System.Native/pal_networking.h

```diff
@@ -60,9 +60,9 @@ typedef enum
     AddressFamily_AF_UNIX = 1,     // System.Net.AddressFamily.Unix
     AddressFamily_AF_INET = 2,     // System.Net.AddressFamily.InterNetwork
     AddressFamily_AF_INET6 = 23,   // System.Net.AddressFamily.InterNetworkV6
-    AddressFamily_AF_NETLINK = 30, // System.Net.AddressFamily.Netlink
-    AddressFamily_AF_PACKET = 31,  // System.Net.AddressFamily.Packet
-    AddressFamily_AF_CAN = 32,     // System.Net.AddressFamily.ControllerAreaNetwork
+    AddressFamily_AF_NETLINK = 65536, // System.Net.AddressFamily.Netlink
```

Comment:
We should add a comment here explaining why we jumped to these higher numbers. Basically summarize the design discussion.

Reply:
should the comment be here? This is mirror of c# enum per comment above. I can possibly add that to the enum it self

Reply:
>This is mirror of c# enum per comment above.

What comment text are you referring to? It doesn't seem obvious to me.  A casual reader will wonder why the numbering jumped from 23 to 65536. We should be more explicit about the explanation, i.e. "the numbering jumped so that it was outside the range of 16 bit number, etc."

Reply:
```
/**
 * Address families recognized by {Get,Set}AddressFamily.
 *
 * NOTE: these values are taken from System.Net.AddressFamily. If you add
 *       new entries, be sure that the values are chosen accordingly.
 */
typedef enum
{
    AddressFamily_AF_UNSPEC = 0,   // System.Net.AddressFamily.Unspecified
    AddressFamily_AF_UNIX = 1,     // System.Net.AddressFamily.Unix
    AddressFamily_AF_INET = 2,     // System.Net.AddressFamily.InterNetwork
    AddressFamily_AF_INET6 = 23,   // System.Net.AddressFamily.InterNetworkV6
    AddressFamily_AF_NETLINK = 65536, // System.Net.AddressFamily.Netlink
    AddressFamily_AF_PACKET = 65537,  // System.Net.AddressFamily.Packet
    AddressFamily_AF_CAN = 65538,     // System.Net.AddressFamily.ControllerAreaNetwork
} AddressFamily;
```

The actual numbers follow value from AddressFamily.cs.

Reply:
Well maybe the extra explaination doesn't need to be here exactly since it references "System.Net.AddressFamily".  But is there enough comments in that file to explain why things jump from 23 to 65536?  And is there enough comments/documentation in the code to prevent a developer 2 years from now from adding a new value say "24" thinking it's ok to fill in that gap (which it isn't).

Reply:
I agree with spirit of the comment. I added more comments. Let me know if you have suggestions on wording.